src/i18n/translations/en/translation.json

```
@@ -1 +1,14 @@
-{}
\ No newline at end of file
+{
+  "404": {
+    "h1": "404",
```

```suggestion
    "statusCode": "404",
```

Could we instead name this after the status code instead of the tag? If we ever change the design of this page and this is no longer an `h1`, then this key gets kinda weird.

---

yep can do. i was thinking using `h1` is generic enough in case that title ever changed.